Sources/UI/Channels/ChannelsViewController.swift

```diff
@@ -123,8 +123,12 @@ open class ChannelsViewController: ViewController {
     
     // MARK: - Channel Cell
     
+    open func dequeueChannelCell(at indexPath: IndexPath) -> ChannelTableViewCell {
```

Comment:
Can you add comments for open methods?
It will be great if we can make a flow in the class comments of sequence of methods.
```
dequeueChannelCell
         |
         ˅
    channelCell
         |
         ˅
        ...
```